dev_tools/scripts/add_license.sh

```diff
@@ -4,12 +4,11 @@ THIS_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" &>/dev/null && pwd )
 # ${THIS_DIR} is expected to be: ${GINKGO_ROOT_DIR}/dev_tools/scripts
 
 # Use local paths, so there is less chance of a newline being in a path of a found file
-cd "${THIS_DIR}/../.."
+cd "${THIS_DIR}/../.." || exit
 GINKGO_ROOT_DIR="."
 
-
-LICENSE_FILE="${GINKGO_ROOT_DIR}/LICENSE"
-GINKGO_LICENSE_BEACON="******************************<GINKGO LICENSE>******************************"
```

Comment:
does the structure allow the beacon?
I am wondering about some scripts that rely on the BEACON detection.

Reply:
Sorry, I didn't realize that other scripts also use the beacon. I will adjust them.